index.html

```diff
@@ -22,10 +22,9 @@
 <section class="highlight-section-blue">
 	<div class="container flex">
 		<div class="text editable">
-			<h2><strong>Solidity v0.8.19 is here.</strong></h2>
-			<p><a href="https://github.com/ethereum/solidity/releases/tag/v0.8.19">Solidity 0.8.19</a> includes a range of improvements. Most importantly, custom operators can now be defined for <a href="https://blog.soliditylang.org/2023/02/22/user-defined-operators/">user-defined value types</a>!
-				It also contains a fix for a long-standing bug that can result in code that is only used in creation code to also be included in runtime bytecode.
-			<br><br>For all details please refer to the <a href="https://blog.soliditylang.org/2023/02/22/solidity-0.8.19-release-announcement/">release announcement</a>. We have also included 6 bugfixes in this release!</p>
+			<h2><strong>Solidity v0.8.20 is here.</strong></h2>
+			<p><a href="https://github.com/ethereum/solidity/releases/tag/v0.8.20">Solidity 0.8.20</a> includes a range of improvements in the via-IR pipeline and improves the list of events exposed in the contract ABI, and, most importantly, support for the Shanghai hard fork!
```

Comment:
```suggestion
			<p><a href="https://github.com/ethereum/solidity/releases/tag/v0.8.20">Solidity 0.8.20</a> includes a range of improvements in the via-IR pipeline and improves the list of events exposed in the contract ABI, and, most importantly, introduces support for the Shanghai hard fork!
```

Or something - there's a verb missing there otherwise

Reply:
Good catch.